test/TesterInternal/StreamingTests/SingleStreamTestRunner.cs

```diff
@@ -510,7 +510,7 @@ private Task ValidatePubSub(Guid streamId, string providerName)
                 activationCount = await consumer.GetNumActivations(this.client);
             }
             var expectActivationCount = 0;
-            logger.Info("Test {0} CheckGrainsDeactivated: {1}ActivationCount = {2}, Expected{1}ActivationCount = {3}", testNumber, str, activationCount, expectActivationCount);
+            logger.Info("Test {testNumber} CheckGrainsDeactivated: {type}ActivationCount = {activationCount}, Expected{type}ActivationCount = {expectActivationCount}", testNumber, str, activationCount, expectActivationCount);
```

Comment:
Is this right? The parameter count doesn't match? In the MembershipOracle change you duplicate the parameter which is used twice.